source/plasma3DClass.py

```diff
@@ -43,33 +47,30 @@ def __init__(self):
 		self.bbZmin = None
 		self.bbZmax = None
 		
+		# Default inputs
+		self.plasma3Dmask = False
+		self.NCPUs = 10
 		self.loadHF = False
 		self.loadBasePath = None
 		
-		self.allowed_vars = ['plasma3Dmask','shot','time','tmax','gFile','itt','response',
+		
+	def allowed_class_vars(self):
```

Comment:
@wingena in allowed_class_vars() there should be a markdown compatible comment that describes each variable.  This comment is not only useful for future users, but also is used directly in the readthedocs to describe the parameters needed in the input file.  If there are units associated with an input variable, they should be defined here.  Here is an example code snippet to get you started:

```
   def allowed_class_vars(self):
      .. Writes a list of recognized class variables to HEAT object
      .. Used for error checking input files and for initialization

       plasma3D Variables:
       ----------------------------

      :plasma3Dmask:  boolean that determines if 3D plasmas should be turned on.  If True, uses M3DC1 equilibrium
      :itt: integer number of toroidal iterations...etc.

etc. etc...
```

I added your plasma3DClass allowedClassVars to the documentation in this commit:  https://github.com/plasmapotential/HEAT/pull/31/commits/d9eabb7bdd8371274927b2a2b4df4ee19e3549e0

This is where the input file parameters from allowedClassVars are rendered:  https://heat-flux-engineering-analysis-toolkit-heat.readthedocs.io/en/latest/inputFile.html



Please go through the allowedClassVars and add a separate line for each variable.  For a more extensive example, see the allowedClassVars function in heatFluxClass.py

Reply:
Thanks, I updated the variable description